### PR TITLE
 perf: debounce updateNodeData

### LINKED ISSUE #2225: Reduce frequency of updates to node data
## Detailed Description

The `UPDATE_NODE_DATA` action gets called a lot in quick succession after creating a new wallet. We call this every time receive an update from the channel graph subscription with updated node data.

## Context

excessive calls to `UPDATE_NODE_DATA` noticed in console after new wallet creation.

## Possible Implementation

We should batch/throttle these in order to prevent excessive re-renders.

### PR DESCRIPTION
**Pull request by korhaliv**
## Description:
Resolve #2225
This PR also introduces easily reusable `debounceMiddleware`.  See code comments for usage example

## How Has This Been Tested?
Manually

## Types of changes:
Perf optimization

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [ ] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: utils/debouncedFuncQueue.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import debounce from 'lodash.debounce'
```
Comment (mrfelton) [inline utils/debouncedFuncQueue.js]: This should now be in the form:
```js
import { debounce } from 'lodash/debounce'
```

Comment (korhaliv) [reply utils/debouncedFuncQueue.js]: updated

Comment (korhaliv) [reply utils/debouncedFuncQueue.js]: should be
`import debounce from 'lodash/debounce'`
:)

Comment (mrfelton) [reply utils/debouncedFuncQueue.js]: doh!

**Pull request merged**